backend/Tasque.WebAPI/Middlewares/ExceptionMiddleware.cs

```diff
@@ -20,11 +23,13 @@ public async Task InvokeAsync(HttpContext httpContext)
             }
             catch (HttpException ex)
             {
+                _logger.LogError("A new http exception has been thrown: {ex.StatusCode}| {ex.Message}", ex.StatusCode, ex.Message);
```

Comment:
Logs are configured to display message level so you don't have to explain what happened, just pass {status code} - {error message}.

Reply:
Fixed it